tree/ntuple/v7/test/ntuple_pages.cxx

```diff
@@ -1,12 +1,27 @@
-#include "gtest/gtest.h"
+#include "ntuple_test.hxx"
 
-#include <ROOT/RPageAllocator.hxx>
-#include <ROOT/RPagePool.hxx>
+TEST(RNTuple, Storage)
```

Comment:
I think we should remove this test entirely, it is fully covered by other, better tests.

Reply:
Done in 1ffbeca